packages/core/src/metrics/MetricsPublisher.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
```

Comment:
@wtrocki What is axios?
Is it some cross platform (cordova, react native & web) http(s) lib?
Is this a 'de-facto' lib for solving this kind of problem?
Is there no native http(s) thing (`fetch` api?) for doing this

Reply:
@david-martin I have done research for the networking layers.

Reply:
@david-martin correct, it's a cross platform http library. The standard is `fetch`, i think the only advantage of axios is only that is has more features and also works on Node.js (not relevant for us though).

Reply:
@pb82 Fetch is not supported seamlessly on web: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

Reply:
@wtrocki Agreed.
Separate issue to figure out a 'cross platform' http lib.
That would depend on what range of devices/browsers we should support.
This may come in handy when doing that https://caniuse.com/#feat=fetch